build.psm1

```diff
@@ -2754,7 +2762,9 @@ function Start-CrossGen {
         )
     }
 
-    foreach ($assemblyName in $psCoreAssemblyList) {
+    $fullAssemblyList = $commonAssembliesForAddType + $psCoreAssemblyList
+
+    foreach ($assemblyName in $fullAssemblyList) {
```

Comment:
@adityapatwardhan as a prerequisite, we need all the dependent IL assemblies in order to generated then ngen ones. So, I will leave it as it.